Paralayout/UIView+Alignment.swift

```diff
@@ -246,6 +270,20 @@ extension UIView {
         )
     }
 
+    /// Move the view to align it within its superview.
+    ///
+    /// - precondition: The receiver must have a superview.
+    ///
+    /// - parameter position: The position in both the receiving view and its `superview` to use for alignment.
+    /// - parameter offset: An additional offset to apply to the receiver.
+    public func alignToSuperview(_ position: Position, offset: UIOffset) {
```

Comment:
Tangential to this PR, I feel like we might want to rename these to `alignWithSuperview` to match `align(_:withSuperviewPosition:)` and `align(_:with:_:)`.

Reply:
Filed #79 to track this